<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SickRecordsPage</class>
 <widget class="QWidget" name="SickRecordsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sick Records</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="sick_records_table"/>
   </item>
   <item>
    <widget class="QLineEdit" name="sick_date">
     <property name="placeholderText">
      <string>Sick Date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="symptoms">
     <property name="placeholderText">
      <string>Symptoms</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_sick_record_btn">
       <property name="text">
        <string>Add Sick Record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_btn">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
